src/Text/Markdown/SlamDown/Halogen/Component.purs

```diff
@@ -432,19 +496,34 @@ renderFormElement config st id label field =
     SD.DropDown msel (Identity ls) →
       pure $
         case msel of
-          M.Nothing → renderDropDown id label ls M.Nothing
+          M.Nothing → renderDropDown ident label ls M.Nothing
           M.Just (Identity sel) →
             let options = if sel `F.elem` ls then ls else L.Cons sel ls
-            in renderDropDown id label options $ M.Just sel
+            in renderDropDown ident label options $ M.Just sel
 
   where
+    renderPicker
+      ∷ PickerType
+      → HTML v m
+    renderPicker pickerType =
+      case pickerType of
+        DatePicker → HH.slot' cpDatePicker label
+          (DatePicker.picker datePickerFormat) unit $ HE.input $
+          \(NotifyChange n) → SDQ.ChangeTextBox label $ SD.Date $ value n
```

Comment:
Can you indent these `NotifyChange` lambdas please? Formatting this way doesn't make it obvious they're part of the previous line.